src/components/data/toolbar/DataDotMenu.js

```diff
@@ -458,7 +458,9 @@ function DataDotMenu(props) {
                 onClose={() => setPathListDlgOpen(false)}
                 onCreatePathList={(id, path) => {
                     setPathListDlgOpen(false);
-                    routeToFile(id, path);
+                    if (id && path) {
```

Comment:
Also, at the very least, the following line should be updated to check for undefined `path` and `id` as well:
https://github.com/cyverse-de/sonora/blob/d714dd6f09347280a868e925a24819426e25a08a/src/components/data/PathListAutomation.js#L103
Such as `onCreatePathList(data?.file?.id, data?.file?.path);`

We'll probably also want to display an alert that the request is processing, and the user will get a notification when it's complete. Like we do when moving files/folders, for example:
https://github.com/cyverse-de/sonora/blob/d714dd6f09347280a868e925a24819426e25a08a/src/components/data/MoveDialog.js#L48-L57

Reply:
This is good and I really appreciate the pointers to how to change some of these things since I haven't been working with the UI that much :sweat_smile: 

I think I've fixed both of these things